gulpfile.js

```diff
@@ -231,11 +231,11 @@ gulp.task('react:modules', function() {
 
 gulp.task('react:extract-errors', function() {
   return merge(
-    gulp.src(paths.react.src).pipe(extractErrors(errorCodeOpts)),
-    gulp.src(paths.reactDOM.src).pipe(extractErrors(errorCodeOpts)),
-    gulp.src(paths.reactNative.src).pipe(extractErrors(errorCodeOpts)),
-    gulp.src(paths.reactTestRenderer.src).pipe(extractErrors(errorCodeOpts))
-  );
+    gulp.src(paths.react.src),
+    gulp.src(paths.reactDOM.src),
+    gulp.src(paths.reactNative.src),
+    gulp.src(paths.reactTestRenderer.src)
+  ).pipe(extractErrors(errorCodeOpts));
```

Comment:
The error code generator was written in a way that it reads the file first, traverses the AST, and appends to the target file. I guess when we invoke it multiple times it causes a race condition and overwrites the same file.

To be honest there should a better way if I'm more knowledgeable in Gulp :). This PR simply changes it back to the way that the generator was used.